### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by mbazhlekova**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# How to test this change?
+
+Please describe the steps to test your changes.
+
+- Step 1
+- Step 2
+- etc
+
+
+# Checklist:
```
Comment (kidk) [inline .github/PULL_REQUEST_TEMPLATE.md:25]: I'd put this higher. Above "How to test this change?"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Please delete options that are not relevant.
+
+- [ ] My commit message follows [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
+- [ ] My code is formatted to [Go standards](https://go.dev/blog/gofmt)
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
```
Comment (kidk) [inline .github/PULL_REQUEST_TEMPLATE.md:34]: Should we add the local test commands for this? `make build`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How to test this change?
+
+Please describe the steps to test your changes.
+
+- Step 1
```
Comment (kidk) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Let's clarify this a little bit more. What are we expecting? A HCL file? Set of commands?

**Review by kidk — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (mbazhlekova) [reply .github/PULL_REQUEST_TEMPLATE.md]: I added a link to the testing section of the README.

**Review by kidk — APPROVED** [review]

**Pull request merged**